package.json

```diff
@@ -25,10 +25,9 @@
     "stylelint-scss": "^3.19.0",
     "webpack": "^5.44.0",
     "webpack-cli": "^4.7.2",
-    "webpack-dev-server": "^3.11.2"
+    "webpack-dev-server": "^3.11.3"
```

Comment:
- [ ] Kindly make sure you install jest testing framework 🙏 . You can read more about it from their [documentation](https://jestjs.io/docs/getting-started)